crd/fleet/fleet-crds.yaml

```diff
@@ -4064,6 +5652,40 @@ spec:
                   - resource
                   type: object
                 type: array
+              endTime:
+                description: EndTime is the time failover reached its final state.
+                format: date-time
+                type: string
+              internal:
+                description: Internal is used by the system controllers. You should
+                  not directly depend on the information in this section.
+                properties:
+                  newPrimary:
+                    description: NewPrimary is the instance that we are attempting
```

Comment:
Replace "we" with "you" -- "we" is restricted for situations when we, as Google, recommend something to the user.